### ISSUE TITLE
On you example filter gender is undefined for the both male and female

### ISSUE THREAD

**Issue by mylastore** [dasDaniel/svelte-table#2]
On you example filter gender is undefined for the both male and female

**Comment by dasDaniel** [comment]
Hello Oscar, thanks for checkout out my library.
Can you clarify what the issue is, I can't reproduce to get an error.

**Comment by mylastore** [comment]
On the sample data sorting by gender on the drop down I see undefined undefined for both male and female options

**Comment by dasDaniel** [comment]
OK, so I don't see that. Can you provide more info?
Browser, OS, steps you used to setup and see test would be helpful.

**Comment by imperator-maximus** [comment]
I can confirm this issue on Mac - all browser. 
I tred     filterOptions: [{name:"male",value:"male"}, {name:"female",value:"female"}] // provide array
instead. It shows now male/female in Dropdown but after selecting it 
there is error 
Unhandled Promise Rejection: TypeError: columnByKey[f].filterValue is not a function. (In 'columnByKey[f].filterValue(r)', 'columnByKey[f].filterValue' is undefined)
on line 27

**Comment by dasDaniel** [comment]
Took me a while t o figure it out, but I see the problem now. I was looking into the example app included and not the `Readme.md`.

I will update the Readme to reflect correct setup right away, then update functionality to allow passing array of strings.

@imperator-maximus , I think the reason you're getting undefined could be due to case match, I'm considering making matches not case sensitive by default.

**Comment by dasDaniel** [comment]
updated the Readme, thanks for notifying me of the issue. To be clear the issue was that the `filterOptions` needs to be an array of key-val pair objects, and that the `filterValue` needs to be present (as-well as `value`, which in this case would be same) I will look into fixing this regression so that it supports array of strings, and defaults to `value`, making `filterValue` optional

```js
    filterValue: v => v.gender, // setup function for filter comparison
    filterOptions: [ // provide array of key value pairs
      {name:"Male", value:"male"},
      {name:"Female", value:"female"}
    ],
```

**Comment by dasDaniel** [comment]
Released v0.1.6

Made `filterValue` optional with `value` as fallback, and `filterOptions` now accepts array of non-object values (strings, numbers, boolean)
